src/Nancy/GlobalizationConfiguration.cs

```diff
@@ -10,20 +12,37 @@ public class GlobalizationConfiguration
         /// <summary>
         /// A default instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        public static readonly GlobalizationConfiguration Default = new GlobalizationConfiguration(supportedCultureNames: new[] { "en-us" });
+        public static readonly GlobalizationConfiguration Default = new GlobalizationConfiguration(supportedCultureNames: new[] { "en-US" }, defaultCulture: "en-US");
 
         /// <summary>
         /// Initializes a new instance of the <see cref="GlobalizationConfiguration"/> class
         /// </summary>
-        /// <param name="supportedCultureNames"></param>
-        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames)
+        /// <param name="supportedCultureNames">An array of supported cultures</param>
+        /// <param name="defaultCulture">The default culture of the application</param>
+        public GlobalizationConfiguration(IEnumerable<string> supportedCultureNames, string defaultCulture = null)

```

Comment:
The more I think about it, shouldn't passing in an empty `supportedCultureNames` be considered an invalid configuration? And if we do that, then the `ThreadCulture` can probably be nuked?

Perhaps the behavior should be that the *default* (that is set by `DefaultGlobalizationConfigurationProvider`) should actually be `CultureInfo.CurrentCulture` so unless you explicitly call `env.Cultures` then we use the OS culture. 

However *if* you actually do call `env.Cultures` then you `explicitly` care about the supported cultures, hence passing in null/empty list should be a `ConfigurationException`

The same behavior for `defaultCulture` should still remain as it is implemented below (optional, ensure it's one of the supported, use the first supported as the fallback default)

Does that not create a much more solid story?